app/scripts/catalogue-sync.ts

```diff
@@ -68,10 +68,8 @@ async function syncDataCatalogue() {
     });
   }
   const previousUpdate = catalogue.lastUpdate?.getTime() || 0;
-
-  const lastUpdateResponse = await fetch(
-    `${GLOBAL_API_URL}/api/v0/catalogue/last-update`,
-  );
+  const catalogUrl = `${GLOBAL_API_URL}/api/v0/catalogue`;
```

Comment:
```suggestion
  const catalogUrl = `${GLOBAL_API_URL}/api/v0/catalogue/i18n`;
```
Why are you still using the old route?

Reply:
we have to keep it without the i18n for the last_updated check. then we add the /i18n when we query for the data.